tests/installation/first_boot.pm

```diff
@@ -52,30 +76,24 @@ sub run() {
             type_password;
             send_key 'ret';
         }
-        handle_login;
     }
-
-    my @tags = qw(generic-desktop);
-    if (check_var('DESKTOP', 'kde') && get_var('VERSION', '') =~ /^1[23]/) {
-        push(@tags, 'kde-greeter');
-    }
-    # GNOME and KDE get into screenlock after 5 minutes without activities.
-    # using multiple check intervals here then we can get the wrong desktop
-    # screenshot at least in case desktop screenshot changed, otherwise we get
-    # the screenlock screenshot.
-    my $timeout        = 600;
-    my $check_interval = 30;
-    while ($timeout > $check_interval) {
-        my $ret = check_screen \@tags, $check_interval;
-        last if $ret;
-        $timeout -= $check_interval;
-    }
-    # the last check after previous intervals must be fatal
-    assert_screen \@tags, $check_interval;
-    if (match_has_tag('kde-greeter')) {
-        send_key "esc";
-        assert_screen 'generic-desktop';
+    else {
+        handle_first_desktop;
+        mouse_hide();    # hide mouse for clean logout needles
+                         # logout to test login on openSUSE https://progress.opensuse.org/issues/13306
```

Comment:
that comment indendation is misleading, please move both comment lines above each statement, recheck and auto-LGTM :-)